io.github.baldeuniversel.am-okay.yml

```diff
@@ -0,0 +1,46 @@
+id: io.github.baldeuniversel.am-okay
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk 
+
+command: am-okay
+
+finish-args:
+    - --filesystem=host
+    - --filesystem=/tmp
+    - --filesystem=/media
+
+modules:
+    - name: am-okay-package
+      buildsystem: simple      
+      sources:
+          - type: git 
+            url: https://github.com/baldeuniversel/am-okay.git 
+            branch: main
```

Comment:
```suggestion
```

Reply:
Is that relevant to change the `branch` ?

Reply:
using branches is not allowed. in that case the build WILL break as soon as the tip of the branch isn't the specified commit. @bbhtt knows. Don't argue.